packages/commercetools/composables/src/getters/orderGetters.ts

```diff
@@ -1,26 +1,46 @@
-import { UserOrderGetters, AgnosticOrderStatus } from '@vue-storefront/interfaces';
-import { Order, OrderState } from './../types/GraphQL';
+import { UserOrderGetters, AgnosticOrderStatus, AgnosticPrice } from '@vue-storefront/interfaces';
+import { Order, OrderState, LineItem, Money, Address } from './../types/GraphQL';
 
 export const getOrderDate = (order: Order): string => order?.createdAt || '';
 
 export const getOrderId = (order: Order): string => order?.id || '';
 
+export const getOrderNumber = (order: Order): string => order?.orderNumber || getOrderId(order);
```

Comment:
how that is different from ID? Did you made sure that this field is not specific to commercetools?

Reply:
A lot of ecommerce platforms have the ID and Order Number separately.

Reply:
```
export const getOrderId = (order: Order): string => order?.id || order?.orderNumber;

```

Reply:
does a lot mean all of them? keep in mind that heleprs are for the UI, are both needed in the UI?